### PR TITLE
Gb/solar module

### PR DESCRIPTION
**Pull request by grantbuster**

### CODE REVIEW

File: sup3r/solar/solar.py [hunk 1]
```diff
@@ -0,0 +1,630 @@
...
+from rex.utilities.fun_utils import get_fun_call_str
+
+from sup3r.utilities import ModuleName
+from sup3r.postprocessing.file_handling import RexOutputs, H5_ATTRS
+
+
+logger = logging.getLogger(__name__)
+
+
+class Solar:
```
Comment (bnb32) [inline sup3r/solar/solar.py:26]: This is nice work!

File: tests/test_solar_module.py [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+                               inclusive='left', freq='1h')
+
+
+@pytest.fixture(scope='module')
+def runner():
+    """Cli runner helper utility."""
+    return CliRunner()
+
+
+def make_cs_ratio_files(td, low_res_times, low_res_lat_lon):
```
Comment (bnb32) [inline tests/test_solar_module.py:42]: move this to test_utils.py? Doesnt really matter though

**Review by bnb32 — APPROVED** [review]
Nice use of farms. Looks great

File: sup3r/solar/solar.py (hunk 1 above)
Comment (grantbuster) [reply sup3r/solar/solar.py:26]: Thanks! feels so good to have a full day without meetings and a blank slate haha.

**Comment by grantbuster** [comment]
Yeah! irrad models are my jam. This worked out great.

**Pull request merged**